### PR TITLE
Add organization README.md

### PR DESCRIPTION
**Pull request by maltoe**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Hey, we are bitcrowd 👋
+
+![An illustration showing a futuristic city hovered by bitcrowd's spaceship](https://bitcrowd.net/assets/header-landscape_1500w-992799f5719297b7904493dc38f57dc640be0fbd1ad8f231ec7c7a53303cf777.jpg)
+
+We build hand-tailored software solutions for small companies and big corporations. If you want to bring your idea to life, come visit us at [our homepage](https://bitcrowd.net)!
+
+### Open Source Software
+
+We're standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have time, we love to give back: By contributing patches to projects of others or by maintaining and improving our own OSS libraries, which you can take a look at here.
```
Comment (ammancilla) [inline profile/README.md]:
```suggestion
We're standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have the time, we love to give back either by contributing to existing OSS libraries or sharing our own ones, which you can take a look at here.
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+## Hey, we are bitcrowd 👋
+
+![An illustration showing a futuristic city hovered by bitcrowd's spaceship](https://bitcrowd.net/assets/header-landscape_1500w-992799f5719297b7904493dc38f57dc640be0fbd1ad8f231ec7c7a53303cf777.jpg)
+
+We build hand-tailored software solutions for small companies and big corporations. If you want to bring your idea to life, come visit us at [our homepage](https://bitcrowd.net)!
+
+### Open Source Software
+
+We're standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have the time, we love to give back either by contributing to existing OSS libraries or sharing our own ones, which you can take a look at here.
```
Comment (planktonic) [inline profile/README.md]:
```suggestion
We’re standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have the time, we love to give back either by contributing to existing OSS libraries or sharing our own ones, which you can take a look at here.
```

Should there be a link to our open source projects (mentioned at the end of the text), or does github render some links around this automatically?

Comment (maltoe) [reply profile/README.md]: It shows the pinned repositories right underneath, hence we wrote this an intro for them. Maybe we should even replace `here` with `below` in the last sentence?

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+## Hey, we are bitcrowd 👋
+
+![An illustration showing a futuristic city hovered by bitcrowd's spaceship](https://bitcrowd.net/assets/header-landscape_1500w-992799f5719297b7904493dc38f57dc640be0fbd1ad8f231ec7c7a53303cf777.jpg)
+
+We build hand-tailored software solutions for small companies and big corporations. If you want to bring your idea to life, come visit us at [our homepage](https://bitcrowd.net)!
+
+### Open Source Software
+
+We’re standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have the time, we love to give back either by contributing to existing OSS libraries or sharing our own ones, which you can take a look at here.
```
Comment (maltoe) [inline profile/README.md]:
```suggestion
We’re standing on the shoulders of giants, as a lot of our software depends on the foundations built by the amazing open source community 💗. Hence, whenever we have the time, we love to give back either by contributing to existing OSS libraries or sharing our own ones, which you can take a look at below.
```

Comment (maltoe) [reply profile/README.md]: @planktonic

Comment (planktonic) [reply profile/README.md]: I see! Yup that looks good 👍

**Review by andreasknoepfle — APPROVED** [review]
👏

**Pull request merged**